RussianFilter/sections/specific.txt

```diff
@@ -3527,6 +3527,7 @@ westnews.com.ua##a.count
 kick.media##a.js-article-sidebar-ad
 кинокрад.net##a.link_casin
 tree.tv##a.new_back_link
+forum.antichat.ru##.funboxWrapper  a > img
```

Comment:
`ul[class^="sam1Bannner"]`
or
`ul[class^="sam"][class*="Bannner"]`
and check existing rules, some are obsolete